testsuite/python/observable_cylindrical.py

```diff
@@ -46,180 +49,153 @@ class TestCylindricalObservable(ut.TestCase):
         'max_z': 5.0,
     }
 
+    v_r = 0.6
+    v_phi = 0.7
+    v_z = 0.8
+
     def tearDown(self):
         self.system.part.clear()
 
-    def swap_axis(self, arr, axis):
-        if axis == 'x':
-            arr = np.dot(
-                tests_common.rotation_matrix([0, 1, 0], np.pi / 2.0), arr)
-        elif axis == 'y':
-            arr = np.dot(
-                tests_common.rotation_matrix([1, 0, 0], -np.pi / 2.0), arr)
-        return arr
-
-    def swap_axis_inverse(self, arr, axis):
-        if axis == 'x':
-            arr = np.dot(
-                tests_common.rotation_matrix([0, 1, 0], -np.pi / 2.0), arr)
-        elif axis == 'y':
-            arr = np.dot(
-                tests_common.rotation_matrix([1, 0, 0], np.pi / 2.0), arr)
-        return arr
-
-    def pol_coords(self):
-        positions = np.zeros((len(self.params['ids']), 3))
-        for i, p in enumerate(self.system.part):
-            tmp = p.pos - np.array(self.params['center'])
-            tmp = self.swap_axis_inverse(tmp, self.params['axis'])
-            positions[
-                i, :] = tests_common.transform_pos_from_cartesian_to_polar_coordinates(tmp)
-        return positions
-
-    def set_particles(self):
-        self.system.part.clear()
-        # Parameters for an ellipse.
-        a = 1.0  # semi minor-axis length
-        b = 2.0  # semi major-axis length
-        # Choose the cartesian velocities such that each particle gets the same
-        # v_r, v_phi and v_z, respectively.
-        self.v_r = .75
-        self.v_phi = 2.5
-        self.v_z = 1.5
-        for i in self.params['ids']:
+    def calc_ellipsis_pos_vel(
+            self, n_part, z_min, z_max, semi_x=1., semi_y=1.):
+        """
+        Calculate positions on an elliptical corkscrew line. Calculate cartesian velocities that lead to a constant velocity in cylindrical coordinates
```

Comment:
here and in other python tests, please wrap docstrings to 80 characters per line (the formatting script is not allowed to wrap strings because adding newlines in a string may change the script behavior)